src/Config.hxx

```diff
@@ -17,7 +18,10 @@ NullableString(const std::string &s) noexcept
 	return s.empty() ? nullptr : s.c_str();
 }
 
+
 struct Config {
+	using IgnoreListMap = std::map<std::string, std::shared_ptr<IgnoreList>>;
```

Comment:
Why is this a `shared_ptr`? And if there really is a good reason to do this, why did you not include `<memory>`?

Reply:
I agree refcounting is not needed.

I will switch the owning pointer to unique_ptr and the non-owning pointer in the ScrobblerConfig to a raw pointer. Please let me know if you disagree with this approach.

Reply:
Explain why you want the `unique_ptr` indirection

Reply:
For automatic memory management. Do you propose to manually allocate and free?

Reply:
No, I don't propose to do that. Why would I want to do that? That would still be an indirection, just an unmanaged one. I don't want the indirection.

Reply:
Oh right, sorry I misunderstood you. Indeed, the map value is stable so we can point to it directly. I will remove the indirection.